src/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -1393,8 +1381,7 @@ public Territory getRocketAttack(final Collection<Territory> candidates, final T
   }
 
   public static int save(final String filename, final GameData m_data) {
-    try (FileOutputStream fos = new FileOutputStream(filename);
-        ObjectOutputStream oos = new ObjectOutputStream(fos);) {
+    try (FileOutputStream fos = new FileOutputStream(filename); ObjectOutputStream oos = new ObjectOutputStream(fos);) {
```

Comment:
@DanVanAtta That is just an auto-format change if you look closely and is unrelated to the changes in this PR. I'd ask that its addressed in a separate PR and that this is PR merged.